app/assets/sass/components/_list.scss

```diff
@@ -0,0 +1,66 @@
+.app-list {
```

Comment:
What's this used for?

Reply:
It's used in the `candidate-feedback.njk`